client/src/utils/systems.test.js

```diff
@@ -33,8 +33,9 @@ describe('systems utility functions', () => {
       'projects',
       systemList,
       projectsListingFixture,
-      'test.site.project.FRONTERA-X'
-    )).toEqual('');
+      'test.site.project.FRONTERA-X',
+      'Test Project Title'
+    )).toEqual('Test Project Title');
```

Comment:
Consider adding the the test where the fallback parameter isn't there as it is still used in some places:
```suggestion
    )).toEqual('Test Project Title');
    expect(findSystemOrProjectDisplayName(
      'projects',
      systemList,
      projectsListingFixture,
      'test.site.project.FRONTERA-X'
    )).toEqual('');
```